e2e/provision/gce_install_sandbox.sh

```diff
@@ -68,20 +65,15 @@ cat "$HOME/.ssh/id_rsa.pub" >>"$HOME/.ssh/authorized_keys"
 
 KVER=$(uname -r)
 
-if ! lsmod | grep -q gtp5g; then
-    [[ -d "$HOME/gtp5g" ]] || git clone --depth 1 -b v0.6.8 https://github.com/free5gc/gtp5g.git "$HOME/gtp5g"
-
-    pushd "$HOME/gtp5g" >/dev/null
-    GCC=gcc
-    if [[ $(uname -v) == *22.04.*-Ubuntu* ]]; then
-        GCC=gcc-12
-    fi
-    if ! command -v $GCC >/dev/null; then
-        sudo apt-get -y install $GCC
+if [[ ${LOAD_GTP5G_MODULE:-false} == "true" ]] && ! lsmod | grep -q gtp5g; then
```

Comment:
Maybe I have to enable by default
```suggestion
if [[ ${LOAD_GTP5G_MODULE:-true} == "true" ]] && ! lsmod | grep -q gtp5g; then
```